docs/pbc/all/miscellaneous/202407.0/tutorials-and-howtos/how-to-extend-the-log-structure-with-additional-data.md

```diff
@@ -0,0 +1,250 @@
+---
+title: "HowTo: Extend the audit log structure with additional data"
+description: Learn how to enhance your audit log structure by adding supplementary data to improve auditing.
+template: howto-guide-template
+---
+
+In this guide, we will walk you through the steps to extend the audit log structure with additional data. 
+As an example a business unit of a logged-in customer for **Yves** application will be added. 
+
+There are two primary methods to extend data in your audit logs:
+* Introduce Processor Plugins: if registered for the corresponding audit log type and application will always be executed and will try to extend the data. 
+See [HowTo: Add a new audit log type](https://documentation.spryker.com/docs/how-to-add-a-new-audit-log-type) for more information.
+* Pass the data to the specific log context: You can pass additional data directly to the specific log context as needed, 
+providing flexibility for different logging scenarios.
+
+## Prerequisites
+
+Before you begin, ensure that the [Spryker Core feature](/docs/pbc/all/miscellaneous/{{page.version}}/install-and-upgrade/install-features/install-the-spryker-core-feature.html)
+is installed. Without it, you will not be able to proceed with the steps outlined in this guide.
+
+## Step-by-step instructions
+
+### Option 1: Introduce Processor Plugins
+
+* Obtain the **RequestStack** service to get access to the customer session:
```

Comment:
As for me, it would be more clear to rephrase it to something like 
"Add **RequestStack** service to the dependency provider of your module to get access to the customer session".